crates/ursa-gateway/src/config.rs

```diff
@@ -107,6 +113,9 @@ impl Default for GatewayConfig {
                 max_size: 200_000_000,  // 200MB
```

Comment:
any plans to increase this?

Reply:
I will create a ticket to track this issue status; thanks for reminding 🙏